Sources/SpeziValidation/ValidationState/ValidationState.swift

```diff
@@ -67,7 +67,7 @@ public struct ValidationState: DynamicProperty {
 extension ValidationState {
     /// A binding to a ``ValidationState``.
     @propertyWrapper
-    public struct Binding {
+    public struct Binding: Sendable {
         private let binding: SwiftUI.Binding<ValidationContext>
```

Comment:
What do you think of constraining the access to `@MainActor` (and properties below). This would make sure that we do not rely on the `SwiftUI.Binding` `Sendable` conformance which not reflect reality.

```suggestion
@MainActor        private let binding: SwiftUI.Binding<ValidationContext>
```

Reply:
@lukaskollmer For context, the `Binding` documentation states this:
```
A binding conforms to Sendable only if its wrapped value type also conforms to Sendable. 
It is always safe to pass a sendable binding between different concurrency domains. 
However, reading from or writing to a binding’s wrapped value from a different concurrency domain may or may not be safe, depending on how the binding was created. 
SwiftUI will issue a warning at runtime if it detects a binding being used in a way that may compromise data safety.
```

Reply:
incorporated this suggestion 👍